packages/core/src/textures/TextureSystem.ts

```diff
@@ -19,6 +19,12 @@ export class TextureSystem extends System
 {
     public boundTextures: BaseTexture[];
     public managedTextures: Array<BaseTexture>;
+    /**
+     * Whether glTexture with int/uint sampler type was uploaded
+     * @member {boolean}
+     * @protected
+     */
+    protected hasIntegerTextures: boolean;
```

Comment:
We don't need to add `@member` and `@protected` in the docs for this. webdoc will infer.